js/battle.js

```diff
@@ -2305,7 +2307,7 @@ var Battle = (function () {
 		if (window.console && console.log) console.log(text);
 	};
 	Battle.prototype.log = function (html, preempt) {
-		var willScroll = (this.logFrameElem.scrollTop() + 60 >= this.logElem.height() + this.logPreemptElem.height() - this.logFrameElem.height());
+		var willScroll = (this.logFrameElem.scrollTop() + 60 >= this.logElem.height() + this.logPreemptElem.height() - this.optionsElem.height() - this.logFrameElem.height());
```

Comment:
It seems that this change made it so that a single mouse-wheel scroll (2.5 lines) no longer prevents autoscroll.